rali/rali_pybind/example/tf_detection_pipeline.py

```diff
@@ -79,12 +104,28 @@ def main():
 	pipe.build()
 	
 	imageIterator =  RALIIterator(pipe)
-	for i, (images_array, bb, labels_array) in enumerate(imageIterator, 0):
-		print("\nIMAGES ARRAY:\n",images_array)
-		print("\nBBOXS ARRAY:\n",bb)
-		print("\nLABELS ARRAY:\n",labels_array)
-			
+	sum = 0
+	for i, (images_array, bboxes_array, labels_array, num_bboxes_array) in enumerate(imageIterator, 0):
+		images_array = np.transpose(images_array, [0, 2, 3, 1])
+		print("RALI augmentation pipeline - Processing batch %d....." % i)
+		for element in list(range(bs)):
+			print("Processing image %d....." % element)
+			features_dict = {
+				"image" : images_array[element],
+				"true_image_shape" : np.array([len(images_array[element]), len(images_array[element,0]), len(images_array[element,0,0])])
+			}
+			labels_dict = {
+				"num_groundtruth_boxes" : num_bboxes_array[element],
+				"groundtruth_boxes" : bboxes_array[element],
+				"groundtruth_classes" : get_onehot(labels_array[element], numClasses),
+				"groundtruth_weights" : get_weights(num_bboxes_array[element])
+			}
+			processed_tensors = (features_dict, labels_dict)
+			print("\nPROCESSED_TENSORS:\n",processed_tensors)
+			sum += 1
 		
+		break
+    print("\n\nPrinted first batch with", sum, "images!")
```

Comment:
Codacy found an issue: [unindent does not match any outer indentation level (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6302037)